docs/new_resources.md

```diff
@@ -49,6 +49,22 @@ $Splat = @{
 }
 ConvertTo-DSC @Splat
 ```
+
```

Comment:
Can we add 

## Microsoft Edge Example

here to clarify?

Reply:
Can you write that as a suggestion block? Not sure I understand

Reply:
Did the latest commit solve this?

Reply:
Yep. Thanks!